src/pages/Spacing.vue

```diff
@@ -0,0 +1,304 @@
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-purple {
+  background-color: #c084fc1a;
+  background-image: linear-gradient(135deg,#a855f780 10%,transparent 0,transparent 50%,#a855f780 0,#a855f780 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-pink {
+  background-color: #f472b61a;
+  background-image: linear-gradient(135deg,#ec489980 10%,transparent 0,transparent 50%,#ec489980 0,#ec489980 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-indigo {
+  background-color: #818cf81a;
+  background-image: linear-gradient(135deg,#6366f180 10%,transparent 0,transparent 50%,#6366f180 0,#6366f180 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-blue {
+  background-color: #60a5fa1a;
+  background-image: linear-gradient(135deg,#3b82f680 10%,transparent 0,transparent 50%,#3b82f680 0,#3b82f680 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-fuchsia {
+  background-color: #e879f91a;
+  background-image: linear-gradient(135deg,#d946ef80 10%,transparent 0,transparent 50%,#d946ef80 0,#d946ef80 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+
+/* Support, yay or nay? They are nice */
+/* Same effect as -m-4 on parent and m-4 on children */ 
+.space-x-4 > * + * {
+  margin-left: 1rem; 
+}
+
+</style>
+<template>
+  <h1>Basic usage</h1>
+  <container>
+    <box class="flex flex-wrap items-start justify-center text-white text-sm font-bold leading-6 -mx-5">
+      <div class="flex items-start">
+        <div class="flex-none px-5">
+          <div class="bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div striped class="h-6 bg-stripes-white striped rounded-t-lg"></div>
+            <div class="p-4">pt-6</div>
+          </div>
+        </div>
+        <div class="flex-none px-5 pt-6">
+          <div class="flex bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="flex-none p-4">pr-4</div>
+            <div class="flex-none w-4 bg-stripes-white"></div>
+          </div>
+        </div>
+      </div>
+      <div class="flex items-start">
+        <div class="flex-none px-5 pt-6">
+          <div class="bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="p-4">pb-8</div>
+            <div class="h-8 bg-stripes-white"></div>
+          </div>
+        </div>
+        <div class="flex-none flex px-5 pt-6">
+          <div class="flex bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="flex-none w-2 bg-stripes-white"></div>
+            <div class="flex-none p-4">pl-2</div>
+          </div>
+        </div>
+      </div>
+    </box>
+  </container>
+  <h2>Code example</h2>
+  <container>
+    <!-- Placeholder for a syntax-higlighter -->
+    <syntax-highlighter>
```

Comment:
Great thought/foresight to include the syntax highligher here!

Happy to discuss what that should be (and build it or help build it together) so we can get something in place!